nixos/modules/programs/command-not-found/command-not-found.pl

```diff
@@ -10,6 +10,23 @@
 
 my $dbPath = "@dbPath@";
 
+if (! -e $dbPath) {
+    print STDERR "$program: command not found\n";
+    print STDERR "\n";
+    print STDERR "command-not-found: Missing package database\n";
```

Comment:
For a user that doesn't know about the `package-not-found` mechanism, this error lacks context.

```suggestion
    print STDERR "command-not-found:  A package providing $program can possible be installed, but a package database is missing to tell for sure.\n";
```
... or the like would probably be more clear

Reply:
Good point, I've added more context in.